yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerIOLauncher.java

```diff
@@ -182,19 +214,28 @@ public void onNext(Frame item) {
         }
     }
 
+    @Override
+    public void afterDisconnect(SlaveComputer computer, TaskListener listener) {
+    }
+
+    @Override
+    public void beforeDisconnect(SlaveComputer computer, TaskListener listener) {
+    }
 
     @Override
     public DockerComputerLauncher getPreparedLauncher(String cloudId, DockerSlaveTemplate dockerSlaveTemplate,
                                                       InspectContainerResponse ir) {
-        return new DockerComputerIOLauncher();
+        final DockerComputerIOLauncher launcher = new DockerComputerIOLauncher();
+        launcher.setJavaPath(getJavaPath());
+        launcher.setJvmOpts(getJvmOpts());
+        return launcher;
     }
 
     @Override
     public void appendContainerConfig(DockerSlaveTemplate dockerSlaveTemplate,
                                       CreateContainerCmd createContainerCmd) throws IOException {
 //        createContainerCmd.withEntrypoint("/bin/bash", "-c", "java -jar /tmp/slave.jar");
-        createContainerCmd.withEntrypoint("sleep");
-        createContainerCmd.withCmd("infinity");
+        createContainerCmd.withCmd("cat");
```

Comment:
cat will be waiting for stdin or stdout, it not just a process that sleeps AFAIR